test/unit/test_api_acl.py

```diff
@@ -91,23 +157,107 @@ def test_update_entry(self):
 
 
+class TestApiExtendedAcls(EapiConfigUnitTest):
+
+    def __init__(self, *args, **kwargs):
+        super(TestApiExtendedAcls, self).__init__(*args, **kwargs)
+        self.instance = pyeapi.api.acl.ExtendedAcls(None)
+        self.config = open(get_fixture('running_config.text')).read()
+
+    def test_get(self):
+        result = self.instance.get('exttest')
+        keys = ['name', 'type', 'entries']
+        self.assertEqual(sorted(keys), sorted(result.keys()))
+        self.assertEqual(result['type'], 'extended')
+
+    def test_get_not_configured(self):
+        self.assertIsNone(self.instance.get('unconfigured'))
+
+    def test_acl_functions(self):
+        for name in ['create', 'delete', 'default']:
+            if name == 'create':
+                cmds = 'ip access-list exttest'
+            elif name == 'delete':
+                cmds = 'no ip access-list exttest'
+            elif name == 'default':
+                cmds = 'default ip access-list exttest'
+            func = function(name, 'exttest')
+            self.eapi_positive_config_test(func, cmds)
+
+    def test_update_entry(self):
+        cmds = ['ip access-list exttest', 'no 10',
+                '10 permit ip 0.0.0.0/32 1.1.1.1/32 log', 'exit']
+        func = function('update_entry', 'exttest', '10', 'permit', 'ip',
+                        '0.0.0.0', '32', '1.1.1.1', '32', True)
+        self.eapi_positive_config_test(func, cmds)
+
+    def test_update_entry_no_log(self):
+        cmds = ['ip access-list exttest', 'no 10',
+                '10 permit ip 0.0.0.0/32 1.1.1.1/32', 'exit']
+        func = function('update_entry', 'exttest', '10', 'permit', 'ip',
+                        '0.0.0.0', '32', '1.1.1.1', '32')
+        self.eapi_positive_config_test(func, cmds)
+
+    def test_remove_entry(self):
+        cmds = ['ip access-list exttest', 'no 10', 'exit']
+        func = function('remove_entry', 'exttest', '10')
+        self.eapi_positive_config_test(func, cmds)
+
+    def test_add_entry(self):
+        cmds = ['ip access-list exttest',
+                'permit ip 0.0.0.0/32 1.1.1.1/32 log', 'exit']
+        func = function('add_entry', 'exttest', 'permit', 'ip', '0.0.0.0',
+                        '32', '1.1.1.1', '32', True)
+        self.eapi_positive_config_test(func, cmds)
+
+    def test_add_entry_no_log(self):
+        cmds = ['ip access-list exttest', 'permit ip 0.0.0.0/32 1.1.1.1/32',
+                'exit']
+        func = function('add_entry', 'exttest', 'permit', 'ip', '0.0.0.0',
+                        '32', '1.1.1.1', '32')
+        self.eapi_positive_config_test(func, cmds)
+
+    def test_add_entry_with_seqno(self):
+        cmds = ['ip access-list exttest',
+                '30 permit ip 0.0.0.0/32 1.1.1.1/32 log', 'exit']
+        func = function('add_entry', 'exttest', 'permit', 'ip', '0.0.0.0',
+                        '32', '1.1.1.1', '32', True, 30)
+        self.eapi_positive_config_test(func, cmds)
+
+
```

Comment:
might be good idea to add a test case with 'host' keyword since the parsing will be a bit different